packages/super-components/components/msc-tail-logs/mock/v6/me-contact.json

```diff
@@ -0,0 +1,14 @@
+{
+  "address": {
+    "country": "FR",
+    "city": "Roubaix",
+    "zip": "59100",
+    "line1": "La rue du Test"
+  },
+  "firstName": "Annie Galou",
+  "id": 11111111,
+  "language": "fr_FR",
+  "lastName": "Lalie Panouc",
+  "legalForm": "individual",
+  "phone": "+33.123456789"
+}
```

Comment:
**suggestion:**  I suppose we use `me-contact.json` in lot of places. Maybe we can move this to a more reusable place.

Reply:
This is just a response example, we should not make multiple components depend on the same mock.
That's like making multiple components depend on the same test : if you need to change something in the test scenario some day you will break another component.
And the mock does not cost anything since it is not included in the production build.